src/aind_data_transfer_models/watchdog/manifest_config.py

```diff
@@ -0,0 +1,84 @@
+"""Job configs for VAST staging or executing a custom script"""
+
+from datetime import datetime
+from typing import Dict, Optional, List
+
+from aind_data_schema_models.platforms import Platform
+from aind_data_schema_models.modalities import Modality
+from pydantic import BaseModel, Field, field_validator, ConfigDict
+
+
+class ManifestConfig(BaseModel):
+    """Configuration for session:
+    based on engineerings lims_scheduler_d manifest files"""
+
+    model_config = ConfigDict(extra="forbid")
+    name: str = Field(
+        ...,
+        description="Unique name for session data assets",
+        title="Unique name",
+    )
+    processor_full_name: str = Field(
+        ..., description="User who processed the data", title="Processor name"
+    )
+    subject_id: int = Field(..., description="Subject ID", title="Subject ID")
+    acquisition_datetime: datetime = Field(
+        description="acquisition datetime in YYYY-MM-DD HH:mm:ss format",
+        title="Acquisition datetime",
+    )
+    schedule_time: Optional[datetime] = Field(
+        default=None,
+        description="Transfer time to schedule copy and upload. If None defaults to trigger the transfer immediately",  # noqa
+        title="APScheduler transfer time",
+    )
+    platform: str = Field(description="Platform type", title="Platform type")
+    capsule_id: Optional[str] = Field(
+        ..., description="Capsule ID of pipeline to run", title="Capsule"
+    )
+    s3_bucket: Optional[str] = Field(
+        default=None, description="s3 endpoint", title="S3 endpoint"
+    )
+    project_name: str = Field(
+        ..., description="Project name", title="Project name"
+    )
+
+    destination: str = Field(
+        ...,
+        description="where to send data to on VAST",
+        title="VAST destination and maybe S3?",
+    )
+    modalities: Dict[str, List[str]] = Field(
+        default={},
+        description="list of ModalityFile objects containing modality names and associated files or directories",  # noqa
```

Comment:
Instead of the `noqa`, you can wrap the stings in () and skip lines. Something like:
```python
    modalities: Dict[str, List[str]] = Field(
        default={},
        description=(
            "list of ModalityFile objects containing modality names and "
            "associated files or directories"
        ),
```

Reply:
Barg! Black removes the parenthesis and moves everything to one line and I didn't catch it with flake8 because I forgot to pull the noqa out 🙄 I will fix this shortly